spec/spec_helper.rb

```diff
@@ -1,3 +1,11 @@
+require 'capybara/rspec'
+
+require 'capybara/poltergeist'
+Capybara.javascript_driver = :poltergeist
+
+# WARN: tilt autoloading 'coffee_script' in a non thread-safe way; explicit require 'coffee_script' suggested.
+require 'coffee_script'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.